.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/TimePeriodParserChs.cs

```diff
@@ -76,6 +83,76 @@ public DateTimeParseResult Parse(ExtractResult er, DateObject refDate)
             return null;
         }
 
+        private DateTimeResolutionResult ParseNight(string text, DateObject referenceTime)
+        {
+            int day = referenceTime.Day,
+                month = referenceTime.Month,
+                year = referenceTime.Year;
+            var ret = new DateTimeResolutionResult();
+            var match = this.config.TimeOfDayRegex.Match(text);
+            
+            if (!GetMatchedTimexRange(text, out string timex, out int beginHour, out int endHour, out int endMinSeg))
+            {
+                return new DateTimeResolutionResult();
+            }
+            
+            ret.Timex = timex;
+            ret.FutureValue = ret.PastValue = new Tuple<DateObject, DateObject>(
+               DateObject.MinValue.SafeCreateFromValue(year, month, day, beginHour, 0, 0),
+               DateObject.MinValue.SafeCreateFromValue(year, month, day, endHour, endMinSeg, endMinSeg)
+               );
+            ret.Success = true;
+            
+            return ret;
+        }
+        
+        private bool GetMatchedTimexRange(string text, out string timex, out int beginHour, out int endHour, out int endMin)
+        {
+            var trimmedText = text.Trim();
+            beginHour = 0;
+            endHour = 0;
+            endMin = 0;
+            
+            if (trimmedText.EndsWith("上午"))

```

Comment:
Ideally all these hardcoded strings should be defined in the YAML file. Can you move them there?

Reply:
Sorry, I don't know how to achieve this since there are the similar hardcoded strings in English.

Reply:
You can take a look at the English YAML file. DateTokenPrefix shows an example of declaring a string there.
Ideally the string hardcoded in other languages should also move. This could be a separate PR.
Thanks.